sql/telemetry_derived/fission_monitoring_crash_v1/query.sql

```diff
@@ -1,15 +1,35 @@
-WITH crash_ping_data AS (
+WITH first_sessions AS (
+    -- these are the sessions where clients were enrolled in the experiment
+    -- since fission is enabled after restart, we want to filter them out from the main pings
+  SELECT
+    session_id
+  FROM
+    `moz-fx-data-shared-prod.telemetry.events`
+  WHERE
+    event_category = 'normandy'
+    AND normalized_channel = 'nightly'
+    AND event_method = 'enroll'
+    AND submission_date = @submission_date
+    AND event_string_value = 'bug-1660366-pref-ongoing-fission-nightly-experiment-nightly-83-100'
+),
+crash_ping_data_unfiltered AS (
   SELECT
     submission_timestamp,
     client_id,
-    -- TODO: update filtering when https://bugzilla.mozilla.org/show_bug.cgi?id=1667426 is finalized
+    payload.session_id,
     CASE
     WHEN
-      mozfun.map.get_key(environment.settings.user_prefs, 'fission.autostart') = 'true'
+      mozfun.map.get_key(
+        environment.experiments,
+        "bug-1660366-pref-ongoing-fission-nightly-experiment-nightly-83-100"
+      ).branch = 'fission-enabled'
     THEN
       'enabled'
     WHEN
-      mozfun.map.get_key(environment.settings.user_prefs, 'fission.autostart') = 'false'
```

Comment:
Please add this preference be output for the `moz-fx-data-shared-prod.telemetry_derived.fission_monitoring_crash` table. Then a cross-check can be performed during analysis to make sure the preferences match.

Reply:
This doesn't need to be done for the `fission_monitoring_main_v1` table, as the `environment.settings.user_prefs` payload is already available.